simulation/environment/GIS.py

```diff
@@ -111,28 +113,38 @@ def __init__(self, api_key, origin_coord, dest_coord, waypoints, race_type, forc
         self.path_gradients = helpers.calculate_path_gradients(self.path_elevations,
                                                                self.path_distances)
 
-
-    @helpers.timeit
     def calculate_closest_gis_indices(self, cumulative_distances):
         """
-        Takes in an array of point distances from starting point, returns a list of 
+        Takes in an array of point distances from starting point, returns a list of
         self.path indices of coordinates which have a distance from the starting point
         closest to the point distances.
 
         :param cumulative_distances: (float[N]) array of distances,
         where cumulative_distances[x] > cumulative_distances[x-1]
-        
+        :param golang: Flag of whether to use faster Go implementation.
```

Comment:
This is confusing since golang itself is not a parameter to this function. It is contained in self so it doesn't need to be listed separately in the documentation.